draft-ietf-quic-transport.md

```diff
@@ -2998,17 +2998,15 @@ The QUIC packet size includes the QUIC header and integrity check, but not the
 UDP or IP header.
 
 Clients MUST ensure that the first Initial packet they send is sent in a UDP
```

Comment:
nit: how about:
"Clients MUST ensure they send the first Initial packet in a UDP datagram that is at least 1200 bytes."